java/src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1310,6 +1313,22 @@ void audioJustification() {
 
                 utils.testValid("audioJustification with and without optional URI argument");
             }
+            
+            /**
+             * Create video justifications with and without optional URIs.  Without a URI, a blank node is created.
+             */
+            @Test
+            void videoJustification() {
+                final Double startTimestamp = 5.0;
+                final Double endTimestamp = 10.0;
+
+                Resource justification = makeVideoJustification(model, utils.getDocumentName(), startTimestamp, endTimestamp, InterchangeOntology.VideoJustificationChannelBoth, system, confidence);
+                makeVideoJustification(model, utils.getDocumentName(), startTimestamp*1.1, endTimestamp*1.1, InterchangeOntology.VideoJustificationChannelSound, system, confidence, utils.getUri("custom-uri-" + ++uriCount));
```

Comment:
@ecurley-nextcentury,

Just curious, what does the *1.1 do  in "startTimestamp*1.1, endTimestamp*1.1,"

Reply:
Haha, nothing. Good catch. I was copying/pasting from audioJustification(). I'll fix it and push a change to remove that.

Reply:
Oops, looks like I did use the timestamps to create multiple justifications. The purpose behind multiplying by 1.1 is to make sure the justifications aren't all pointing to the same locations. Now that I realize that I _did_ actually make multiple justifications, I'm not going to make any change. Sorry for the confusion